src/bokeh/models/ui/ui_element.py

```diff
@@ -66,6 +66,17 @@ class StyledElement(Model):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
 
+    html_attributes = Dict(String, String, default={}, help="""
+    Allows to configure HTML attributes on the underlying HTML element.
+    """)
+
+    html_id = Nullable(String, default=None, help="""
+    Sets the ``id` attribute of the underlying HTML element.
+
+    This is a shorthand for a common HTML attribute. Alternatively ID can be
+    set using ``html_attributes`` property.
```

Comment:
```suggestion
    This is a shorthand for the common HTML "id" attribute. Alternatively the id can be
    set in the ``html_attributes`` dictionary.
```
What happes if id is set in both places?

Reply:
`html_id` takes preference.

Reply:
After additional changes, this applies to other properties as well (e.g. `style`).